apps/docs-snippets/src/guide/encoding/encode-and-decode.test.ts

```diff
@@ -82,16 +82,16 @@ describe('encode and decode', () => {
     // #endregion encode-and-decode-4
 
     // #region encode-and-decode-5
-    // #import { FunctionInvocationResult, AbiCoder, ReceiptType, TransactionResultReturnDataReceipt, arrayify};
+    // #import { AbiCoder, ReceiptType, TransactionResultReturnDataReceipt, arrayify, buildSubmitResult };
 
     // Get result of the transaction, including the contract call result. For this we'll need
     // the previously created invocation scope, the transaction response and the script
-    const invocationResult = await FunctionInvocationResult.build(
-      [invocationScope],
-      response,
-      false,
-      script
-    );
+    const invocationResult = await buildSubmitResult({
+      funcScope: invocationScope,
+      isMultiCall: false,
```

Comment:
@danielbate, you are right; we can remove the isMultiCall.

The motivation behind this code refactoring was to simplify how the response of a contract/script call is handled without introducing any breaking changes.

Even though I had to mark this PR as a breaking change because the `InvocationResult` class was removed, I am maintaining the structure of the returned object.